github/filecoin-project.yml

```diff
@@ -297,9 +294,6 @@ repositories:
     advanced_security: false
     allow_update_branch: false
     archived: false
-    collaborators:
-      admin:
-        - snadrus
```

Comment:
A bunch of these look like some kind of cleanup or dedupe, not removing active members? are diffs creating changes or reflecting an existing reality or just deduping individuals who already have access as a team, e.g. this one `snadrus` should be in `Curio` team which has admin?

This is good, but it does make it slightly harder to review. No strong opinion on whether it should be split off to a separate PR, just noting this fact.

Reply:
Well as I read more, I'm not really sure what I would be reviewing by looking at this! So yes, a change summary of "X will lose access to Y" would certainly help.